client/src/components/DataTable.tsx

```diff
@@ -45,30 +46,41 @@ export const DataTable: React.FC<Props> = ({
             })}
           </tr>
         </thead>
+        
         <tbody>
-          {dataRows.map((row) => {
-            const [id, ...data] = row;
-            if (typeof id !== "string" && typeof id !== "number") {
-              console.error(
-                "Please provide an ID of type 'string' or 'number' as the first entry of each row."
+          {dataRows.length === 0?(
+            
+            <tr>
+              <td colSpan={columnHeaders.length} className="text-center py-4 text-2xl font-sans " >
+                NO DATA AVAILABLE
```

Comment:
This part looks like it's for the other Issue handled in #153. I suggest keeping this PR focused solely on the past events column.